runtime/binding-kafka-grpc/src/main/java/io/aklivity/zilla/runtime/binding/kafka/grpc/internal/stream/KafkaGrpcRemoteServerFactory.java

```diff
@@ -1353,6 +1373,8 @@ private GrpcClient(
             this.initialId = supplyInitialId.applyAsLong(routedId);
             this.replyId = supplyReplyId.applyAsLong(initialId);
             this.correlater = new KafkaCorrelateProxy(originId, resolveId, replyTo, server.condition, this);
+            this.buffer = new ExpandableDirectByteBuffer();
+            this.buffer.putStringWithoutLengthAscii(0, META_PREFIX);
```

Comment:
Do we need a separate expandable buffer per client, or can we move this to factory (per core) instead for reuse?